packages/graphql_client/lib/models/nodes.dart

```diff
@@ -1040,6 +1040,8 @@ class NodeEdgeInfo {
   }
 }
 
+
+@reflector
```

Comment:
reflector package not working with nested stuff in classes.

Reply:
![image](https://github.com/user-attachments/assets/f96b74a1-7961-4afc-b7f6-65c3c7c216c1)


works fine.